examples/example_direct_gaussian.py

```diff
@@ -21,36 +21,36 @@
 
 fig, ax = plt.subplots(1,2)
 
-ax[0].set_title('Forward transform of a Gaussian')
-ax[1].set_title('Inverse transform of a Gaussian')
-
-ax[0].plot(ref.r, ref.abel, 'b', label='Analytical transform')
-
-recon = direct_transform(ref.func, dr=ref.dr, direction="forward",
-                        correction=True, backend='C')
-
-ax[0].plot(ref.r, recon , '--o',c='red', label='direct')
-
-recon = direct_transform(ref.func, dr=ref.dr, direction="forward",
-                        correction=True, backend='Python')
-
-ax[0].plot(ref.r, recon , ':d', c='k', label='direct naive')
-
-
-ax[1].plot(ref.r, ref.func, 'b', label='Original function')
-
-recon = direct_transform(ref.abel, dr=ref.dr, direction="inverse",
-                         correction=True)
-
-ax[1].plot(ref.r, recon , '--o', c='red', label='direct')
-
-recon = direct_transform(ref.abel, dr=ref.dr, direction="inverse",
+# forward Abel transform
+reconC = direct_transform(ref.func, dr=ref.dr, direction="forward",
+                          correction=True, backend='C')
+reconP = direct_transform(ref.func, dr=ref.dr, direction="forward",
```

Comment:
We are now pretty confident that the Direct-C and Direct-P methods produce identical results (it's a unit test now). So, perhaps it's more interesting to plot one of the Direct methods with and without the correction?

Reply:
Ok. Why use `correction=False`?

![plot_example_direct_gaussian](https://user-images.githubusercontent.com/10932229/34549803-978e573e-f15f-11e7-9f01-efcb8b8546dc.png)